easybuild/easyconfigs/m/magma/magma-1.6.2-CrayGNU-5.2.40.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "ConfigureMake"
+
+name = 'magma'
+version = '1.6.2'
+
+homepage = 'http://icl.cs.utk.edu/magma/'
+description = """The MAGMA project aims to develop a dense linear algebra library similar to LAPACK 
+    but for heterogeneous/hybrid architectures, starting with current Multicore+GPU systems."""
+
+toolchain = {'name': 'CrayGNU', 'version': '5.2.40'}
+toolchainopts = {'pic': True, 'dynamic': True,  'openmp': True, }
```

Comment:
well, @pforai suggested to use the toolchainopts, but fpic is still needed on make.inc for nvcc